guides/common/modules/proc_migrating-to-newly-deployed-satellite.adoc

```diff
@@ -0,0 +1,57 @@
+[id="migrating-to-newly-formed-{ProjectServer}]_{context}"]
+= Migrating to a Newly Formed {ProjectServer}
+
+*Before you begin*
+
+* Install a minimal {RHEL} 8 instance on the target machine with the capacity to store backup files.
+* Do not install any {RHEL} 8 software groups or third-party applications.
+
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/index[Performing a standard {RHEL} installation].
+
+== Creating a backup of {ProjectServer} or {SmartProxyServer} on {RHEL} 7
+
+Before you perform a fresh installation of the {ProjectServer} or {SmartProxyServer} on the {RHEL} 8 system, back up your {ProjectServer} or {SmartProxyServer} data on the {RHEL} 7 system by creating an offline backup.
+
+If you recently created an offline backup, you can perform an incremental backup to update the existing backup.
+
+*Procedure*
+
+* Perform an offline or incremental backup on the source machine:
+** To perform an offline backup, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html/administering_red_hat_satellite/backing-up-satellite-server-and-capsule-server#performing-a-full-backup-of-satellite-or-capsule_admin[Performing a Full Backup of {ProjectServer} or {SmartProxyServer}].
```

Comment:
It looks like this should be available in the "administering Foreman" guide.

Reply:
@maximiliankolb Can you please elaborate on what you mean by "Please ensure there is only one heading per file and each heading has an anchor. Provide links by either using "docURLs" or "xref". See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L3." w/r/t the link?  I'm not sure how the link you posted is related.  ty

Reply:
Your source file currently contains multiple headings: You start (correctly) with `= My level1 Heading`, and then also (incorrectly) add `== My level2 Heading`.

Please ensure that on have exactly one heading per file.

Use docURLs where appropriate: the admin guide is part of the upstream docs, meaning you can link to it using an attribute: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L3
`#performing-a-full-backup-of-satellite-or-capsule_admin` is probably all you want between the DocURL and title.